mautic/mautic
# Allow to hide a Theme in the Mautic GUI without uninstalling it. Mautic5

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [No]
| New feature/enhancement? (use the a.x branch)      | [Yes]
| Deprecations?                          | [No]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [No] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Allow Themes to be hidden in the theme-Selection when creating an email

Idea:
- Have a toggle in the theme’s config.json to hide the theme in the GUI
- So the Mautic GUI should not offer the hidden themes in https://<mymautic>/s/emails/new
    

#### Steps to test:
- Upload the theme 
- Go to emails
- Create new email
- Check if theme is visible in the overview(it should be if isVisibleInGUI is true)
- Go to CLI and manually change the toggle in the config.json to hide the theme(`cd themes/<name_of_theme> && nano config.json` after change isVisibleInGUI to false)
- Go to emails
- Create new email or edit already existing
- Check if theme is now hidden
